components/Contributors.js

```diff
@@ -10,19 +10,23 @@ export default function Contributors() {
     { name: "Levon", githubLink: "https://github.com/itfibonacci" },
     { name: "Karen", githubLink: "https://github.com/karleee" },
     { name: "James", githubLink: "https://github.com/ngjamesng" },
-    { name: "Megan", githubLink: "https://github.com/mcdevittbass"}
+    { name: "Megan", githubLink: "https://github.com/mcdevittbass" },
```

Comment:
Wanna add your name?

Reply:
Now I know to check if it's yarn.lock or package-lock.json on the project, before I start! I forgot to add my name, I'll push that now too. Thanks, Sam!